content/blog/2023/07/16/2023-07-16-security-scan-probe-blog.adoc

```diff
@@ -50,4 +50,4 @@ During this project, I accomplished the following:
 
 == Links
 
-For more information or if you have any questions, please visit the official GSoC 2023 project link:/projects/gsoc/2023/projects/add-probes-to-plugin-health-score/[Adding Probes to  "Plugin Health Score"] description page.
+For more information or if you have any questions, please visit the official GSoC 2023 project link:../../../../projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc[Adding Probes to  "Plugin Health Score"] description page.
```

Comment:
```suggestion
For more information or if you have any questions, please visit the official GSoC 2023 project link:/projects/gsoc/2023/projects/add-probes-to-plugin-health-score/[Adding Probes to  "Plugin Health Score"] description page.
```
The modified link does not direct anywhere, is there a reason why you modified the links this way? The way it is written originally works and goes to the project page as intended.

Reply:
The links weren't working in my local with the old syntax. That is why I changed it.

I just checked the published blogs and it works fine. 

I think I am missing something here.

Reply:
Links generally should *not* work in GitHub, you either have to run the site locally with `make run` or check on the preview page https://deploy-preview-6653--jenkins-io-site-pr.netlify.app/blog/2023/09/03/jsr305-probe-blog/ (note that URLs on the site do not end with `.adoc`)

Reply:
IntelliJ IDE renders the `.adoc` syntax so I expect it to render even the URLs. That's where I was wrong.